src/aiq/runtime/session.py

```diff
@@ -86,7 +87,8 @@ def context(self) -> AIQContext:
     @asynccontextmanager
     async def session(self,
                       user_manager=None,
-                      request: Request = None,
+                      request: Request | None = None,
+                      websocket_user_message: WebSocketUserMessage | None = None,
```

Comment:
Not a fan of having the websocket as an argument here. Lets just manually specify the conversation ID.